src/components/Form.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Form.css';
+
+export class Form extends React.Component {
+  state = {
+    task: '',
+    userName: '',
+    user: '',
+    userError: false,
+    taskError: false,
+  };
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const { task, userName, user } = this.state;
+    const { addUser } = this.props;
+
+    if (!task || !userName) {
+      this.setState({
+        userError: true,
+        taskError: true,
+      });
+
+      return;
+    }
+
+    addUser(user, task);
+    this.setState({
+      task: '',
+      userName: '',
+      user: '',
+      userError: false,
+      taskError: false,
+    });
```

Comment:
mb setState before addUser, cause now I can't add new user after the error has occurred (event after I've fixed it)